ios/brave-ios/Sources/BraveWallet/Crypto/FungibleAssetView.swift

```diff
@@ -5,15 +5,76 @@
 
 import Preferences
 import SwiftUI
+import BraveCore
 
-struct PortfolioAssetView: View {
+// View used to display `FungibleAssetView` as a button.
+struct FungibleAssetButton: View {
+  let asset: AssetViewModel
+  /// If a container should be shown to around unavailable balance banner
+  /// for Bitcoin asset row (when there is a pending balance)
+  let shouldShowContainerForBitcoin: Bool
+  let currencyFormatter: NumberFormatter
+  @Binding var bitcoinBalanceDetails: BitcoinBalanceDetails?
+  let action: (BraveWallet.BlockchainToken) -> Void
+  
+  var body: some View {
+    // Avoid button inside a button for `UnavailableBTCBalanceView`
+    if asset.btcBalances.compactMap({ $0.value[.pending] }).reduce(0.0, +) > 0 {
```

Comment:
Not sure what this line does ( :) ), but could you please confirm negative pending balance is supported?
For example it's negative when I send BTCs to someone, but transaction is not confirmed yet.

Reply:
This `compactMap` maps all available pending balance (all accounts), then `reduce` calculates sum. Checks for > 0, but need to update to != 0 to show balance for negative pending. Good catch 👍.